kasa/deviceconfig.py

```diff
@@ -142,39 +121,17 @@ def from_values(
                 + f"{encryption_type}.{login_version}"
             ) from ex
 
-    @staticmethod
-    def from_dict(connection_type_dict: dict[str, Any]) -> "DeviceConnectionParameters":
-        """Return connection parameters from dict."""
-        if (
-            isinstance(connection_type_dict, dict)
-            and (device_family := connection_type_dict.get("device_family"))
-            and (encryption_type := connection_type_dict.get("encryption_type"))
-        ):
-            if login_version := connection_type_dict.get("login_version"):
-                login_version = int(login_version)  # type: ignore[assignment]
-            return DeviceConnectionParameters.from_values(
-                device_family,
-                encryption_type,
-                login_version,  # type: ignore[arg-type]
-                connection_type_dict.get("https", False),
-            )
 
-        raise KasaException(f"Invalid connection type data for {connection_type_dict}")
+class _DoNotSerialize(SerializationStrategy):
```

Comment:
I'm wondering if there's a nicer way to flag an attribute not to be exported / to be set as null? I think it is a fairly common occurrence, so perhaps mashumaro has something built-in to be able to avoid a custom strategy?

Reply:
Alas I haven't been able to find an easier way. I'll open a ticket on mashumaru.